package.json

```diff
@@ -29,7 +29,7 @@
     "report-coverage": "cat ./coverage/lcov.info | codecov",
     "validate": "npm t && npm run check-coverage",
     "commit": "git-cz",
-    "install": "opt --out ghooks-install --exec \"node ./bin/install\"",
+    "install": "opt --out ghooks-install --exec \"[ ! -d \"./dist\" ] && npm run build; node ./bin/install\"",
```

Comment:
When cloning / fetching from git using a shell on Windows, I guess people use Git bash or the like?
Because when running `npm install` on Win's default cmd line prompt `cmd.exe` it will possibly fail due to this new statement :disappointed: